README.md

```diff
@@ -1,2 +1,175 @@
-# chaos-scheduler
-Schedule your Chaos!! 
+
+# Litmus chaos-scheduler for injecting chaos experiments on Kubernetes
+  
+Litmus chaos scheduler is used by Kubernetes application developers and SREs to inject chaos into the applications 
+and Kubernetes infrastructure in a managed fashion. Its objective is to make the process of validation and 
+hardening of application workloads on Kubernetes easy by automating the execution of chaos experiments. A sample chaos 
+injection workflow could be as simple as:
+
+- Install the Litmus infrastructure components (RBAC, CRDs), the scheduler & Experiment custom resource bundles via helm charts
+- Annotate the application under test (AUT), enabling it for chaos
+- Create a ChaosEngine custom resource tied to the AUT, which describes the experiment list to be executed 
+
+Benefits provided by the Chaos scheduler include: 
+
+- Scheduled batch Run of Chaos
+- Standardised chaos experiment spec 
+- Categorized chaos bundles for stateless/stateful/vendor-specific
+- Test-Run resiliency 
+- Ability to chaos run as a background service based on annotations
+
+## What is a chaos scheduler and how is it built?
+
+The Chaos scheduler is a Kubernetes scheduler, which are nothing but custom-controllers with direct access to Kubernetes API
+that can manage the lifecycle of certain resources or applications, while always trying to ensure the resource is in the "desired
+state". The logic that ensures this is commonly called "reconcile" function.
+
+The Chaos scheduler is built using the popular [Operator-SDK](https://github.com/operator-framework/operator-sdk/) framework, 
+which provides bootstrap support for new scheduler projects, allowing teams to focus on business/operational logic. 
+
+The Litmus Chaos scheduler helps reconcile the state of the ChaosEngine, a custom resource that holds the chaos intent 
+specified by a developer/devops engineer against a particular stateless/stateful Kubernetes deployment. The scheduler performs
+specific actions upon CRUD of the ChaosEngine, its primary resource. The scheduler also defines secondary resources (the engine 
```

Comment:
ChaosSchedule

Reply:
Secondary resource is ChaosEngine CR here not the runner pod nad monitor service.